### PR TITLE
feat: Recording page

### PR DESCRIPTION
**Pull request by BusraTemizyurek**
- Minor design changes on waiting page.
- Default gauge is drawn for the cases orientation data is not received immediately.
- Recording page ui design (static)
image

- ui update for no location access case:
- 
image

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -44,4 +44,62 @@ body {
     height: 100%;
     background-image: url(./images/bg2.jpg);
     background-position: top;
+}
+
+.text-white {
```
Comment (altinokdarici) [inline index.css:49]: is there a bootstrap css class for this?

File: src/calibration-content.ts [hunk 2]
```diff
@@ -2,10 +2,10 @@ class CalibrationContent {
     private readonly _modalContent: HTMLElement;
     private readonly _waitingContent: HTMLElement;
     private readonly _canvas: HTMLCanvasElement;
-    private readonly _orientation: Orientation;
+    private readonly _orientation: OrientationManager;
```
Comment (altinokdarici) [inline src/calibration-content.ts:5]: rename this to `_orientationManager`

File: src/main-page.ts [hunk 3]
```diff
@@ -38,22 +40,36 @@ class MainPage implements Page {
...
-        this._recordButton?.classList.remove("d-none");
-
+    private async onClickModalReady() {
         if (this._orientation.lastOrientation) {
             const initialOrientation = {
                 alpha: this._orientation.lastOrientation.alpha ?? 0,
                 beta: this._orientation.lastOrientation.beta ?? 0,
                 gamma: this._orientation.lastOrientation.gamma ?? 0
             }
+            console.log(initialOrientation);
```
Comment (altinokdarici) [inline src/main-page.ts:50]: do you really need this logging?

Comment (BusraTemizyurek) [reply src/main-page.ts:50]: I will delete it after I finish with data management

**Review by altinokdarici — APPROVED** [review]

**Pull request merged**